operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -381,14 +381,15 @@ func TestManifestGenerateIstiodRemote(t *testing.T) {
 		g.Expect(objs.kind(gvk.CustomResourceDefinition.Kind).nameEquals("adapters.config.istio.io")).Should(BeNil())
 		g.Expect(objs.kind(gvk.CustomResourceDefinition.Kind).nameEquals("authorizationpolicies.security.istio.io")).Should(Not(BeNil()))
 
+		const istiodServiceName = "istiod"
```

Comment:
nit: the constant could be defined outside of the for loop; we shouldn't need to update the value for every iteration

Reply:
Done. Thanks.